lib/connect-backbone-to-react.js

```diff
@@ -0,0 +1,147 @@
+const hoistStatics = require('hoist-non-react-statics');
+const { Component, PropTypes, createElement } = require('react');
+const debounceFn = require('lodash.debounce');
+const isArray = require('lodash.isarray');
+
+function getDisplayName(name) {
+  return `connectBackboneToReact(${name})`;
+}
+
+function defaultMapModelsToProps(models) {
+  return Object.keys(models).reduce((acc, modelKey) => {
+    const model = models[modelKey];
+    acc[modelKey] = model.toJSON();
+    return acc;
+  }, {});
+}
+
+module.exports = function connectBackboneToReact(
+  mapModelsToProps,
+  options = {}
+) {
+  if (typeof mapModelsToProps !== 'function') {
+    mapModelsToProps = defaultMapModelsToProps;
+  }
+
+  const {
+    debounce = false,
+    events = {},
+    modelTypes = {},
+  } = options;
+
+  function getEventNames(modelName) {
+    let eventNames = events[modelName];
+
+    // Allow turning off event handlers by setting events to false.
+    if (eventNames === false) {
+      return [];
+    }
+
+    if (!isArray(eventNames)) {
+      eventNames = [];
+    }
+
+    if (eventNames.length === 0) {
```

Comment:
Oh I remember! If a user sets the value of the event to an empty array this guarantees that the 'all' event will be added. Which in retrospect is really bad behavior. 

i.e.

```javascript
const options = {
  events: {
    user: []
  },
};
```

That would be very unexpected behavior. Going to fix that.